content/02.introduction.md

```diff
@@ -0,0 +1,19 @@
+# Introduction
+
+1. Biomedcial language is constantly changing
+	1. Scientists make new discoveries
+	2. Old technologies are being revamped
+	3. More reasons for language changes
+2. Diachronic studies aims to capture these types of changes through time
+   1. Insert papers for language changes in general
+      1. doi:10.1007/s00799-019-00271-6
+      2. doi:10.1142/9789811232701_0011
+      3. arxiv:1605.09096
+      4. arxiv:1806.03537
+      5. arxiv:1606.02821
+      6. doi:10.18653/v1/N18-1044 
+3. Gap in knowledge: 
+	1. work has focused majorly on non-biomedical text
+	2. biomedical related work is only on abstract and titles
+	3. Work has yet to focus on individual tokens and themes.
+4. Goal is to show longitudinal trends for biomedical term changes.
```

Comment:
```suggestion
4. Goal is to examine longitudinal trends for biomedical term changes.
```

Reply:
Don't forget in here your methodological contribution to estimate uncertainty with variable-size training datasets in each year by training multiple models and examining variability across them.